From: =?utf-8?q?Martin_Stegh=C3=B6fer?= <martin@steghoefer.eu>
Date: Wed, 29 Oct 2014 20:06:34 +0100
Subject: Make it clear in documentation that -f needs a previous -d.

Bug-Debian: https://bugs.debian.org/359948
Forwarded: https://trac.xiph.org/ticket/1679

The -f option to specify an output file name works only if a file device has previously been specified using the -d option. Some parts of the documentation don't make it very explicit that -f needs -d and that the -d option has to precede the -f option.
---
 ogg123/cmdline_options.c | 2 +-
 ogg123/ogg123.1          | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/ogg123/cmdline_options.c b/ogg123/cmdline_options.c
index d663cc6..619b4f5 100644
--- a/ogg123/cmdline_options.c
+++ b/ogg123/cmdline_options.c
@@ -140,7 +140,7 @@ int parse_cmdline_options (int argc, char **argv,
 	    exit(1);
 	  }
 	} else {
-	  status_error(_("=== Cannot specify output file without specifying a driver.\n"));
+	  status_error(_("=== Cannot specify output file without previously specifying a driver.\n"));
 	  exit (1);
 	}
 	break;
diff --git a/ogg123/ogg123.1 b/ogg123/ogg123.1
index 160a876..7b434c6 100644
--- a/ogg123/ogg123.1
+++ b/ogg123/ogg123.1
@@ -69,7 +69,8 @@ Specify output device.  See
 .B DEVICES
 section for a list of devices.  Any number of devices may be specified.
 .IP "-f filename, --file filename"
-Specify output file for file devices.  The filename "-" writes to standard
+Specify output file for a file device previously specified with
+--device.  The filename "-" writes to standard
 out.  If the file already exists,
 .B ogg123
 will overwrite it.
